assets/scss/modules/hero.scss

```diff
@@ -11,9 +11,28 @@
   }
 }
 
-.strike::before,
-  .strike::after {
+.line {
+  background-image: url("../img/line.png");
+  margin-left: 12.5rem;
+  min-height: 2px;
+  width: 37.5rem;
+}
+
+.call {
+  border: 1px solid $picton-blue;
+  border-radius: 5px;
+  font-size: 18px;
+  min-height: 4.0625rem;
```

Comment:
Properties should be ordered border, border-radius, font-size, margin, max-width, min-height